ciao-4.11/contrib/lib/python3.5/site-packages/ciao_contrib/cda/data.py

```diff
@@ -334,14 +340,16 @@ def is_type(self, types):
         """Given a list of file types, returns True if
         the file is one of these types."""
         for ft in types:
-            if ft == self.filetype: return True
+            if ft == self.filetype:
+                return True
         return False
 
     def is_format(self, formats):
         """Given a list of file formats, returns True if
         the file is one of these formats."""
         for ff in formats:
```

Comment:
Also unrelated to this PR:

Could this just be

```python
    return ff in self.fileformat
```
?

Reply:
It's been so long since I wrote this (I think it's mine) that I'll have to review the code to see what on earth it's doing!

Reply:
Done